src/praisonai-train/tests/unit/test_remote_dispatch.py

```
@@ -0,0 +1,191 @@
+
+    def test_llm_without_remote_settings_does_not_dispatch(self, tmp_path, monkeypatch):
+        monkeypatch.chdir(tmp_path)
+        config = _config(tmp_path, {"model_name": "unsloth/tiny", "dataset": "d.json"})
+        # Stop before the heavy import; the point is only that nothing was sent.
+        monkeypatch.setattr(train_cmd, "import_code_module",
+                            lambda *_a, **_k: (_ for _ in ()).throw(ImportError("no")),
+                            raising=False)
+        try:
+            train_cmd.train_llm(config=config)
+        except Exception:
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)